graylog2-web-interface/src/components/authentication/hooks.js

```diff
@@ -0,0 +1,34 @@
+// @flow strict
```

Comment:
Love it :heart_decoration: . But avoid having a `hooks` file. Do for every hook its own file. But perhaps a hooks folder?

Reply:
Good point, having a look at comparable cases, I named the file `useActiveBackend.js`. I would create a `hooks` directory if we need to add more hooks.